biz.aQute.resolve/src/biz/aQute/resolve/RunResolution.java

```diff
@@ -73,10 +73,28 @@
 	 */
 	public static RunResolution resolve(Project project, Processor actualProperties,
 		Collection<ResolutionCallback> callbacks) {
+		return resolve(project, actualProperties, callbacks, new ResolverLogger());
```

Comment:
I think the last argument need to be `null`. Otherwise the ResolverLogger is not closed.